src/Hopper/Internal/LoweredCore/ClosureConvertedANF.hs

```diff
@@ -269,11 +267,10 @@ data SymbolRegistryCC =
 
 lookupClosureCodeId :: SymbolRegistryCC -> ClosureCodeId-> Either String ClosureCodeRecordCC
 lookupClosureCodeId (SymbolRegistryCC _thk closMap _vals) codeid =
-      maybe  (Left $ "failed closure code lookup " ++ show codeid) (Right) $
+      maybe (Left $ "failed closure code lookup `" ++ show codeid ++ "`") Right $
         Map.lookup codeid closMap
+
 lookupThunkCodeId :: SymbolRegistryCC -> ThunkCodeId-> Either String ThunkCodeRecordCC
 lookupThunkCodeId (SymbolRegistryCC thkmap _closMap _vals) thud =
-      maybe (Left $ "failed thunk code lookup " ++ show thud ) (Right) $
+      maybe (Left $ "failed thunk code lookup `" ++ show thud ++ "`") Right $
```

Comment:
+1 thanks for the reminder. will look at 

* colors: https://hackage.haskell.org/package/annotated-wl-pprint-0.7.0 and  
* annotations: https://hackage.haskell.org/package/ansi-wl-pprint-0.6.7.3/docs/Text-PrettyPrint-ANSI-Leijen.html